storage/innobase/xtrabackup/doc/source/how_xtrabackup_works.rst

```diff
@@ -10,7 +10,11 @@ This works because |InnoDB| maintains a redo log, also called the transaction lo
 
 The above is the backup process. Next is the prepare process. During this step, |Percona XtraBackup| performs crash recovery against the copied data files, using the copied transaction log file. After this is done, the database is ready to restore and use.
 
-The above process is implemented in the |xtrabackup| compiled binary program. The |innobackupex| program adds more convenience and functionality by also permitting you to back up |MyISAM| tables and :term:`.frm` files. It starts |xtrabackup|, waits until it finishes copying files, and then issues ``FLUSH TABLES WITH READ LOCK`` to prevent further changes to |MySQL|'s data and flush all |MyISAM| tables to disk. It holds this lock, copies the |MyISAM| files, and then releases the lock.
+The above process is implemented in the |xtrabackup| compiled binary program. The |innobackupex| program adds more convenience and functionality by also permitting you to back up |MyISAM| tables and :term:`.frm` files. It starts |xtrabackup|, waits until it finishes copying files, and then issues ``FLUSH TABLES WITH READ LOCK`` to prevent further changes to |MySQL|'s data and flush all |MyISAM| tables to disk. It holds this lock, copies the |MyISAM| files, and then releases the lock. 
+
+.. note:: 
+
+  |Percona XtraBackup| will use `Backup locks <https://www.percona.com/doc/percona-server/5.6/management/backup_locks.html#backup-locks>`_ where available as a lightweight alternative to ``FLUSH TABLES WITH READ LOCK``. This feature is available in |Percona Server| 5.6+. |Percona XtraBackup| uses this automatically to copy non-InnoDB data to avoid blocking DML queries that modify InnoDB tables. When backup locks are supported by the server, |xtrabackup| will first copy InnoDB data, run the ``LOCK TABLES FOR BACKUP`` and copy the |MyISAM| tables and :term:`.frm` files. After that |xtrabackup| will use and MDL lock to block all operations that might change either binary log position or ``Exec_Master_Log_Pos`` or ``Exec_Gtid_Set`` (i.e. master binary log coordinates corresponding to the current SQL thread state on a replication slave) as reported by ``SHOW MASTER/SLAVE STATUS``. |xtrabackup| will then finish copying the REDO log files and fetch the binary log coordinates. After this is completed |xtrabackup| will unlock the binary log and tables. 
 
```

Comment:
lets replace "use and MDL lock" with "LOCK BINLOG FOR BACKUP". OK to merge after this is done.